libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFOAuthPlugin/SalesforceOAuthPlugin.m

```diff
@@ -60,18 +60,6 @@ + (NSDictionary *)authErrorDictionaryFromError:(NSError *)error authInfo:(SFOAut
 // ------------------------------------------
 @implementation SalesforceOAuthPlugin
 
-/**
- This is Cordova's default initializer for plugins.
- */
-- (CDVPlugin *)initWithWebView:(UIWebView *)theWebView
```

Comment:
```initWithWebView``` is deprecated and gone in ```4.0```. ```pluginInitialize``` is the new way to add plugin specific functionality.